client/src/api/connectJSON.js

```diff
@@ -61,3 +61,45 @@ export async function deleteWholeWeek() {
     method: "DELETE",
   });
 }
+
+export async function postGroceryList() {
+  const response = await fetch("api/shoppingList/merge");
+  const translate = response.json();
+  return translate;
+}
+
+export async function checkShoppingList() {
+  const shoppingResponse = await fetch(`/api/shoppingList`);
+  const listStatus = await shoppingResponse.json();
+  if (listStatus) {
+    return listStatus;
+  }
+  return false;
+}
+
+export async function addShoppingItem(Quantity, Unit, Grocery) {
```

Comment:
In you code base, you have PascalCase for components. I would rename it:

```suggestion
export async function addShoppingItem(quantity, unit, grocery) {
```